op-service/rpc/cli.go

```diff
@@ -27,12 +28,18 @@ func CLIFlags(envPrefix string) []cli.Flag {
 			Value:   8545,
 			EnvVars: opservice.PrefixEnvVar(envPrefix, "RPC_PORT"),
 		},
+		&cli.BoolFlag{
+			Name:    EnableAdminFlagName,
+			Usage:   "Enable the admin API (experimental)",
```

Comment:
```suggestion
			Usage:   "Enable the admin API",
```
This is no longer experimental. Probably still subject to change, but we do rely on it.

Reply:
done

https://github.com/ethereum-optimism/optimism/pull/7482/commits/1bc388eca88d5b5e69622643bbfc45d86c8bea6a